src/main/java/nextstep/subway/domain/ShortestPathFindAlgorithm.java

```diff
@@ -0,0 +1,7 @@
+package nextstep.subway.domain;
+
+import java.util.List;
+
+public interface ShortestPathFindAlgorithm<V, T, D extends Number> {
```

Comment:
```suggestion
public interface ShortestPathFinder<V, T, D extends Number> {
```
알고리즘을 클래스명에 표현할 필요가 있을까요?
알고리즘이라는 정보를 통해 다른 개발자는 어떤 장점을 얻을 수 있을까요?